qa/L0_java_memory_growth/MemoryGrowthTest.java

```diff
@@ -69,14 +74,16 @@ protected static class DeleteDeallocator extends TRITONSERVER_Server implements
         System.err.println(msg);
       }
 
-      System.err.println("Usage: java " + Simple.class.getSimpleName() + " [options]");
+      System.err.println("Usage: java " + MemoryGrowthTest.class.getSimpleName() + " [options]");
```

Comment:
nit: `getSimpleName()`?

Reply:
This is a Java function to get the class name (java.lang.Class.getSimpleName()), not part of our code. I can see how that'd be confusing, though I don't think this can be helped.